src/SConscript.env

```diff
@@ -166,22 +166,22 @@ if 'msvc' in COMMAND_LINE_TARGETS:
 
     env.Alias('msvc', msvc)
 
-# Print the build flags (useful if the flags have been cached, ie. if you just run "scons"
+# print(the build flags (useful if the flags have been cached, ie. if you just run "scons")
```

Comment:
Nope, english comment here also, nos commented code.

Reply:
Ahhh thanks for the review. This is where sed bites you. I submitted the PR after getting home late last night just to do it. I will clean it up.

It /should/ remain compatible with older versions of scons. Python 2.7 handles the print() function fine, but with the import from __future__ it complain if the keyword version is used.